Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -17,7 +17,47 @@ extension LogtoCore {
         let issuer: String
     }
 
-    static func fetchOidcConfig(endpoint: String, completion: @escaping (OidcConfigResponse?, Error?) -> Void) {
-        Utilities.httpGet(endpoint: endpoint, completion: completion)
+    static func fetchOidcConfig(
+        useSession session: NetworkSession = URLSession.shared,
```

Comment:
the name `useSession` looks like a boolean? what about just `session`

Reply:
discussed offline. good for now.